ddtrace/sampler.py

```diff
@@ -234,33 +232,38 @@ def __init__(
         # Use default sample rate of 1.0
         super(DatadogSampler, self).__init__()
 
-        if default_sample_rate is None:
+        if default_sample_rate is None or not ddconfig._apm_tracing_enabled:
```

Comment:
nit: Just a thought, maybe it could be easier to follow if we do all this under
```
if ddconfig._apm_tracing_enabled:
    [...]
else:
    [...]
```